lib/bitcoin/ErrorCode.ts

```diff
@@ -25,5 +25,6 @@ export default {
   TransactionNumberBlockNumberTooLarge: 'transaction_number_block_number_too_large',
   TransactionNumberTransactionIndexInBlockTooLarge: 'transaction_number_transaction_index_in_block_too_large',
   ValueTimeLockInPendingState: 'value_time_lock_in_pending_state',
-  VersionManagerVersionStringNotFound: 'version_manager_version_string_not_found'
+  VersionManagerVersionStringNotFound: 'version_manager_version_string_not_found',
+  NormalizedFeeCalculatorBlockNotFound: 'normalized_fee_calculator.block_not_found'

```

Comment:
Keep style consistent
```suggestion
  NormalizedFeeCalculatorBlockNotFound: 'normalized_fee_calculator_block_not_found'
```